discovery/discovery_test.go

```diff
@@ -84,20 +77,19 @@ func TestDeadLock(t *testing.T) {
 	assert.Nil(err, "Should not be error")
 	assert.Len(infos, 1, "Should return one orchestrator")
 	assert.Equal("transcoderfromtestserver", infos[0].RemoteInfo.Transcoder)
+	wgWait(&wg)
```

Comment:
do we need to check the returned bool?

Reply:
good idea, checked in a6110ecaa5840d0c1e78b3c4cd26ea52da076aa1